py3-azure-identity.yaml

```diff
@@ -0,0 +1,48 @@
+# Generated from https://pypi.org/project/azure-identity/
+package:
+  name: py3-azure-identity
+  version: 1.15.0
+  epoch: 0
+  description: Microsoft Azure Identity Library for Python
+  copyright:
+    - license: MIT License
```

Comment:
This should just be 'MIT', which is the spdx short identifier:
https://spdx.org/licenses/MIT.html